packages/cli/src/utils/project.ts

```diff
@@ -0,0 +1,42 @@
+import { Project } from '@contember/cli-common'
+import { Schema } from '@contember/schema'
+import * as esbuild from 'esbuild'
+import { join, dirname } from 'path'
+import { NodeVM } from 'vm2'
+import * as fs from 'fs/promises'
+
+export const loadSchema = async (project: Project): Promise<Schema> => {
+	const tempFile = join(process.cwd(), process.env.CONTEMBER_TEMP_SCHEMA_FILE ?? '_contember_schema.js')
+	try {
+		await fs.mkdir(dirname(tempFile), { recursive: true })
+		await esbuild.build({
```

Comment:
if you call it with `write: false` it should given you the output as part of promise result
https://github.com/evanw/esbuild/blob/34899aaa1d76acd3b4adc5a50e125b4e8803a6e0/lib/shared/types.ts#L451

Reply:
https://esbuild.github.io/api/#write